docs/source/whatsnew/1.1.0.txt

```diff
@@ -0,0 +1,165 @@
+Release 1.1.0
+-----------
+
+:Release: 1.1.0
+:Date: February 10, 2016
+
+This release is meant to provide zipline support for pandas 0.18, as well as several bug fixes, API changes, and many
+performance changes.
+
+Highlights
+~~~~~~~~~~
+
+This bumps up pandas to 0.18.1, numpy to 1.11.1, empyrical to 0.2.2 and downgrades scipy 0.17
+
+Enhancements
+~~~~~~~~~~~~
+
+- Makes the minute bar read catch NoDataOnDate exceptions if dates are not in the calendar (:issue:`1488`)
+
+- Switches cachetools.LRUCache with lru-dict LRU Cache (:issue:`1491`)
+
+- Adds truncate method to BcolzMinuteBarWriter (:issue:`1499`)
+
+- Bumps up to pandas 0.18.1 and numpy 1.11.1 (:issue:`1339`)
+
+- Adds an estimates quarter loader for Pipeline (:issue:`1396`)
+
+- Creates an in-memory restricted list (:issue:`1487`)
+
+- Adds `last_available{session, minute}`` args to DataPortal (:issue:`1528`)
+
+- Adds `SpecificAssets` filter (:issue:`1530`)
+
+- Adds _UnionRestrictions for combining multiple Restrictions (:issue:`1487`)
+
+- Adds the ability for an algorithm to request the current contract for a future chain (:issue:`1529`)
+
+- Adds `chain` field to current and supporting methods in DataPortal and OrderedContracts (:issue:`1538`)
+
+- Adds history for continuous futures (:issue:`1539`)
+
+- Adds adjusted history for continuous future (:issue:`1548`)
+
+- Adds roll style which takes the volume of future contract into account (:issue:`1556`)
+
+- Adds a better error message when calling zipline API functions (:issue:`1593`)
+
+- Adds logic that split adjusts estimates (:issue:`1557`)
+
+- Adds MACD, MA, and AnnVol as built-in factors (:issue:`1588`)
+
+- Allows custom chunk iterators for attach_pipeline (:issue:`1617`)
+
+- Adds `order_batch` to the trade blotter (:issue:`1596`)
+
+- Adds vectorized lookup_symbol (:issue:`1627`)
+
+- Solidifies equality comparisons for SlippageModel classes (:issue:`1657`)
+
+- Adds pricing fixture cleanups (:issue:`1667`)
+
+Experimental Features
+~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+   Experimental features are subject to change.
+
+None
+
+Bug Fixes
+~~~~~~~~~
+
+- Changes str to string_types to avoid errors when type checking unicode and not str type. (:issue:`1315`)
+
+- Algorithms default to quantopian-quandl bundle when no data source is specified (:issue:`1479`) (:issue:`1374`)
+
+- Catches all missing data exceptions when computing dividend ratios (:issue:`1507`)
+
+- Fixes blaze query in `ffill_query_in_range` to fix issues in the events loader (:issue:`1530`)
+
+- Creates adjustments based on ordered assets instead of a set (:issue:`1547`)
+
+- Fixes blaze pipeline queries for asof_date (:issue:`1608`)
```

Comment:
The issue was when users queried for the `asof_date` column explicitly, maybe clarify this a little.